code/modules/clothing/clothing.dm

```diff
@@ -56,6 +56,24 @@
 		if(M.putItemFromInventoryInHandIfPossible(src, H.held_index))
 			add_fingerprint(usr)
 
+/obj/item/reagent_containers/food/snacks/clothing
+	name = "oops"
+	desc = "If you're reading this it means I messed up. This is related to moths eating clothes and I didn't know a better way to do it than making a new food object."
+	list_reagents = list("nutriment" = 1)
+	tastes = list("dust" = 1, "lint" = 1)
+
+/obj/item/clothing/attack(mob/M, mob/user, def_zone)
+	if(user.a_intent != INTENT_HARM && ismoth(M))
+		var/obj/item/reagent_containers/food/snacks/clothing/clothing_as_food = new
+		clothing_as_food.name = name
+		var/eaten = clothing_as_food.attack(M, user, def_zone)

```

Comment:
```
if(clothing_as_food.attack(M, user, def_zone))
	take_damage(15)
qdel(clothing_as_food)
```

Reply:
I can fix that in a bit

Reply:
It's not exactly a fix as it's not a bug, it's just removal of a redundant variable

Reply:
That's what I meant